test/studies/isx/PERFEXECOPTS

```diff
@@ -1 +1 @@
---mode=weakISO --useSubTimers=true
+--mode=weakISO --useSubTimers=true --n 8388608
```

Comment:
Thanks for the explanation -- I was thinking that this magic number was preserving the old problem size rather than trying to find a new XC-reasonable problem size.  As we discussed, setting the multiplier to 1 seems like a simpler way to run a problem size that's been reasonable so far while also providing trivial consistency with what we've been doing.